xbmc/threads/Thread.h

```diff
@@ -124,7 +124,7 @@ class CThread
   // -----------------------------------------------------------------------------------
 
   ThreadIdentifier m_ThreadId;
-  ThreadOpaque m_ThreadOpaque;
+  ThreadOpaque m_ThreadOpaque = { 0 };
```

Comment:
This isn't entirely correct. On windows this is a HANDLE which is a pointer type